InteractiveHtmlBom/ecad/fusion_eagle.py

```diff
@@ -0,0 +1,440 @@
+                    if c not in self.pcbdata['font_data']:
+                        self.pcbdata['font_data'][c] = wl
+                if mirrored:
+                    self.pcbdata['drawings']['silkscreen']['B'].append(dwg)
+                else:
+                    self.pcbdata['drawings']['silkscreen']['F'].append(dwg)
+
+    @staticmethod
+    def _segments_to_polygon(segs):
+        polygon = []
+        for vertex in segs.iter('vertex'):
+            polygon.append([float(vertex.attrib['x']), -float(vertex.attrib['y'])])
+        return polygon
+
+    def _add_zone(self, poly, net):
+        dest = self.pcbdata['zones']['F'] if poly.attrib['layer'] == '1' else self.pcbdata['zones']['B']
+        if poly.tag == 'polygonpour':
+            segs = poly.find('polygonfilldetails').find('polygonshape').find('polygonoutlinesegments')
+        else:
+            segs = poly
+        zone = {'net': net,
+                'polygons': []}
+        zone['polygons'].append(self._segments_to_polygon(segs))
+        dest.append(zone)
+
+    def parse(self):
+        ext = splitext(self.file_name)[1]
+        if ext.lower() == '.fbrd':
+            with zipfile.ZipFile(self.file_name) as myzip:
+                brdfilename = [fname for fname in myzip.namelist() if splitext(fname)[1] == '.brd']
+                with myzip.open(brdfilename[0]) as brdfile:
+                    return self._parse(brdfile)
+        elif ext.lower() == '.brd':
+            with open(self.file_name, 'r', encoding='utf-8') as brdfile:
+                return self._parse(brdfile)
+
+    def _parse(self, brdfile):
+        brdxml = ElementTree.parse(brdfile)
+        root = brdxml.getroot()
+        board = root.find('drawing').find('board')
+        plain = board.find('plain')  # contains stuff not associated with a net (silkscreen, holes)
+        elements = board.find('elements')
+        signals = board.find('signals')
+        mv = [el.attrib['value'] for el in root.iter('param') if el.attrib['name'] == 'rlMinViaOuter'][0]
+        mv_val = float(''.join(d for d in mv if d in digits))
+        mv_units = (''.join(d for d in mv if d in ascii_lowercase))
+        if mv_units == 'mm':
+            self.min_via_w = mv_val * 0.001
+        elif mv_units == 'mil':
+            self.min_via_w = mv_val * 0.0254
+        else:
+            self.logger.error(f"Unsupported units {mv_units} on rlMinViaOuter")
+
+        components = []
+
+        # Edges & silkscreen (partial)
+        for el in plain.iter():
+            self._add_drawing(el)
+        # identify board bounding box based on edges
+        board_outline_bbox = BoundingBox()
+        for drawing in self.pcbdata['edges']:
+            self.add_drawing_bounding_box(drawing, board_outline_bbox)
+        if board_outline_bbox.initialized():
+            self.pcbdata['edges_bbox'] = board_outline_bbox.to_dict()
+
+        # Signals --> nets, tracks, zones
```

Comment:
Net data and tracks/zones should be included only if corresponding options are enabled in config.